packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -180,17 +187,20 @@ export default class SearchBox extends Base {
       />
       ${this.clearButton
         ? html`
-            <button @click="${this.clearSearch}">Clear</button>
+            <button
+              class="sfx-search-box__button sfx-search-box__button--close"
```

Comment:
I must repeat my earlier point on this one. I think we should have a discussion about HTML class conventions before committing to this monstrosity. Ahem. /wrant